### PR TITLE
Board class 2

### PR DESCRIPTION
**Pull request by Coralruschak**
Board Class:
Added render method

Board Tests:
Tests added for render method
refactored #valid_placement? overlapping test with correct coordinate information
could use some more edge cases and perhaps new titles for tests? Let's tackle that if we have time to refactor !

### CODE REVIEW

File: lib/board.rb [hunk 1]
```diff
@@ -27,8 +27,8 @@ def valid_coordinate?(coordinate)
     end
 
     def place(ship, coordinates)
-        coordinates.each do |coordinate|
-            @cells[coordinate].place_ship(ship)
+        coordinates.each do |coordinate| 
+             @cells[coordinate].place_ship(ship)
```
Comment (willfox0409) [inline lib/board.rb:31]: I guess we were wrong, we can leave the #valid placement? and #place methods exclusive to each other in the Board class. I think it's in the Game class that they will start to communicate with each other !

File: spec/board_spec.rb [hunk 2]
```diff
@@ -88,9 +88,44 @@
...
+            @board.place(@cruiser, ["A1", "A2", "A3"])
+            @board.cells["C2"].fire_upon
+
+            expect(@board.render).to eq("  1 2 3 4 \nA . . . . \nB . . . . \nC . M . . \nD . . . . \n")
+            expect(@board.render(true)).to eq("  1 2 3 4 \nA S S S . \nB . . . . \nC . M . . \nD . . . . \n")
+        end
+
+        it 'displays that a ship has sunk' do
+            @board.place(@cruiser, ["A1", "A2", "A3"])
+            @board.cells["A1"].fire_upon
```
Comment (willfox0409) [inline spec/board_spec.rb:123]: Cool, I like these test titles, I think they are nice and concise! We can think of some edge-cases we potentially missed once we get into the game-play portion.

**Comment by willfox0409** [comment]
Looks great, I'm going to merge, Iteration 2 complete!
